dsl/src/main/java/tech/pegasys/peeps/node/genesis/GenesisConfig.java

```diff
@@ -26,6 +26,8 @@ public GenesisConfig(final long chainId) {
     this.chainId = chainId;
   }
 
+  public abstract void setSmartContractTransition(final int blockNumber, final String address);
```

Comment:
This doesn't make sense for Clique and ethhash. This really a iBFT & QBFT genesis config item. Is there a need to have this on the base class?